packages/devtools_app/test/flame_chart_test.dart

```diff
@@ -4,6 +4,8 @@
 
 // @dart=2.9
 
+// ignore_for_file: avoid_redundant_argument_values
```

Comment:
why do we need this ignore?

Reply:
Because this library is not migrated yet. Not migrated libraries complain about passing nulls to required parameters that are null by default.